### ISSUE TITLE
State change causes animation to pause

### ISSUE THREAD

**Issue by nishmeht7** [chenqingspring/react-lottie#43]
# Issue

I'm rendering a loading animation, while my results are being fetched. Right before the results appear, the animation hangs, until the new results appear, and the animation is unmounted.

# Expected Behaviour

The animation should keep on loading without a pause, and the results should appear whenever they do, but without having the animation paused.

I was previously using a circular progress material-ui component which is simply a circular rotating loader and the loader would keep looping, and the results would appear, without the user noticing that the loader unmounted.

Has anyone come across such an issue? I would appreciate any input that could help me debug.

**Comment by AntwanSherif** [comment]
Hi @nishmeht7 , Did you find a solution?

**Comment by nishmeht7** [comment]
@AntwanSherif No, I did not. I was considering adding it to a project but didn't go ahead due to that issue.
